Python/Util/pyBitVector.cpp

```diff
@@ -273,4 +273,11 @@ PyObject* pyBitVector_FromBitVector(class hsBitVector& vec) {
     return (PyObject*)bv;
 }
 
+hsBitVector* pyBitVector_AsBitVector(PyObject* value)
```

Comment:
This should not be done at all. The other PyHSPlasma classes provide methods for setting and clearing bit flags instead of implementing a has-a member.